f8a_worker/utils.py

```diff
@@ -663,3 +663,15 @@ def decorated(func, *args, **kwargs):
 
 # get not hidden files from current directory
 # print(list(get_all_files_from('.', file_filter=lambda a: not startswith(a, ['.']))))
+
+
+def url2git_repo(url):
+    """Convert URL to git repo URL and force use HTTPS."""
+    if url.startswith('git@'):
+        url = url[len('git@'):]
+        url = url.split(':')
+        if len(url) != 2:
+            raise ValueError("Unable to parse git repo URL '%s'" % str(url))
+        return 'https://{}/{}'.format(url[0], url[1])
+    else:
+        return url
```

Comment:
Could you place it next to `parse_gh_repo()` and add a small unit test, please ?